rados/write_op_test.go

```diff
@@ -0,0 +1,232 @@
+package rados
+
+import (
+	"github.com/stretchr/testify/assert"
+)
+
+// timeStamp generates a dummy Timespec value.
+func timeStamp() Timespec {
+	// Future TODO (maybe?) - vary the value?
+	return Timespec{342334800, 0}
+}
+
+func (suite *RadosTestSuite) TestWriteOpCreate() {
+	suite.SetupConnection()
+
+	op := CreateWriteOp()
+	defer op.Release()
+	op.Create(CreateIdempotent)
+	err := op.Operate(suite.ioctx, "TestWriteOpCreate", OperationNoFlag)
```

Comment:
it might come in future tests, but there need to be Operate() calls with invalid parameters

Reply:
Sure. If you don't mind I can do a pass to improve coverage in a later PR. This one is already pretty large.